controllers/bucket_controller.go

```diff
@@ -690,7 +690,7 @@ func (r *BucketReconciler) garbageCollect(ctx context.Context, obj *sourcev1.Buc
 		}
 		if len(delFiles) > 0 {
 			r.eventLogf(ctx, obj, eventv1.EventTypeTrace, "GarbageCollectionSucceeded",
-				fmt.Sprintf("garbage collected %d artifacts", len(delFiles)))
+				fmt.Sprintf("garbage collected %d artifacts", len(delFiles)/2))
```

Comment:
Instead of diving by `2`, I would not count the garbage collected locks as items (but see them as "part of").

Reply:
okay then lets change the log statement to "garbage collected %d files"?

Reply:
Still think this would be confusing to a user as the locks are not a thing they are really aware of.